src/main/java/com/fasterxml/jackson/databind/AnnotationIntrospector.java

```diff
@@ -1149,6 +1150,13 @@ public String[] findEnumValues(Class<?> enumType, Enum<?>[] enumValues, String[]
         return names;
     }
 
+    /**
+     * @since 2.15
+     */
+    public String[] findEnumValues(Class<?> enumType, Enum<?>[] enumValues, String[] names, AnnotatedClass annotatedClass){
```

Comment:
Let's add first argument of `MapperConfig<?>` as well -- I have tried adding this to give access to config settings as 3.0 passes that to all methods (and I've retrofitted some old methods in 2.x too).
This assuming it's available to be called.

Also, as usual, `AnnotationIntrospectorPair` needs to be changed to propagate calls.